### ISSUE TITLE
Unable to connect with wifi on 2.2 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [guerrerolor/android-x86#241]
```
What steps will reproduce the problem?
1.Unable to start wifi using any 2.2 iso. Also says "error" 
2. Models tested as follows Dell 2110, Dell D620, Dell e6400, IBM T43, Asus EP 
1001ha
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?


Please provide any additional information below.

Any ideas on how to troubleshoot this, sorry my linux skills are not great. 
Basically I cant get wifi to work on any laptop with any 2.2 iso. As soon as it 
connected it just says "error"
```

Original issue reported on code.google.com by `marculo...@gmail.com` on 8 Feb 2011 at 6:53
